### DISCUSSION TITLE
Install Windows 10

### DISCUSSION THREAD

**Discussion by hekelud** [opendcs/opendcs#102] [Q&A] [answered]
I am trying to download opendcs on my computer as a single user but I keep getting stuck up on the instructions. It says that for windows you should be able to click on the installer icon but I cannot find an opendcs.jar in the download, or anything that resembles that. I tried running the build.xml with ant as well and have been having issues.
In the README it says 
"to build the file opendcs.jar run the following 
`ant jar`
If you want to build the installer run
`ant opendcs` "

But I am confused on what this would pull from to create the file. Maybe I am just looking in the wrong location for all of this and it is a quick fix. Does anyone have any suggestions?

**Comment by agilmore2** [comment]
Are you trying to compile or install the software. The ant command is for
building.

For installers, look at the releases on the right sidebar.

Andrew

**Comment by MikeNeilson** [comment, marked as answer]
Specifically the file with -ot- in it. that's the installer jar.

If you are trying to build and specifically install your built version, the output is in the stage/ directory. Specifically the opendcs-ot-<version>.jar file.

**Reply by MikeNeilson** [reply]
Sorry did I accidentally mark that as the answer or undo what someone else did?

**Reply by MikeNeilson** [reply]
nevermind, finally found the history.
